app/views/users/two_factor_authentication_setup/show.html.erb

```diff
@@ -17,12 +17,12 @@
       <%= twofa_key %>
     </div>
     <div class="govuk-body">
-      2. Enter the six digit code generated by the authentication app on your phone.
+      2. <label for="code-field">Enter the six digit code generated by the authentication app on your phone.</label>
     </div>
     <div class="govuk-body">
       <%= form_with url: users_two_factor_authentication_setup_path, method: :put do |f| %>
         <%= f.hidden_field :otp_secret_key, value: @otp_secret_key %>
-        <%= f.govuk_text_field :code, width: 5, placeholder: "123456", maxlength: 6, label: nil %>
+        <%= f.govuk_text_field :code, id: "code-field", width: 5, placeholder: "123456", maxlength: 6, label: nil %>
```

Comment:
Is this maybe better handled in the form builder? Like this: https://govuk-form-builder.netlify.app/introduction/labels-captions-hints-and-legends/#output-html-setting-label-content-with-a-proc